toolkits/global/packages/global-lists/scss/50-components/_list-description.scss

```diff
@@ -0,0 +1,39 @@
+.c-list-description {
+	font-size: $global-list-description--font-size;
+	font-family: $font-family-sans;
+	line-height: $global-list-description--line-height;
+	margin-bottom: 0;
+	width: 100%; // fix text breaking layout in IE10
+
+	@if $global-list-description--is-interface-link {
```

Comment:
👌 This is going to be `true` for the springernature brand as well, because it will inherit from default, is that desired behaviour?

Reply:
Well so far we have no nested link AFAIK, let me ask James, in case this would happen what he would prefer